### PR TITLE
771 make sample load changes to meet new sample spec

### PR DESCRIPTION
**Pull request by ryangrundy7**
Updated the sample loader to match the sample frame spec. I've removed arids and added print_batch and estab_uprn.

## Test
- generate sample and validate it
- Try and load the sample into the database
- Run with other prs 

[trello](https://trello.com/c/zm2Ssvnn/771-make-sample-load-changes-to-meet-new-sample-spec-13)

### CODE REVIEW

**Comment by ryangrundy7** [comment]
[Make sample load changes to meet new sample spec (13)](https://trello.com/c/zm2Ssvnn/771-make-sample-load-changes-to-meet-new-sample-spec-13)

**Review by NickGrantX — APPROVED** [review]
ATs passing. LGTM 👍

File: validate_sample.py [hunk 1]
```diff
@@ -30,9 +30,8 @@ class SampleValidator:
 
     def __init__(self):
         self.schema = {
-            'ARID': [mandatory(), max_length(21), unique()],
-            'ESTAB_ARID': [mandatory(), max_length(21)],
-            'UPRN': [mandatory(), max_length(12), numeric()],
+            'UPRN': [mandatory(), max_length(12), numeric(), unique()],
```
Comment (AdamHawtin) [inline validate_sample.py]: Are we sure it's valid to make UPRN `unique`? I don't think this is correct, rows like CE units can have the same UPRN.

**Review by AdamHawtin — CHANGES_REQUESTED** [review]

File: tests/test_validate_sample.py [hunk 2]
```diff
@@ -28,10 +28,10 @@ def test_validate_sample_success(self):
         # Then
         self.assertEqual(validation_failures, [])
 
-    def test_validate_sample_duplicate_arids(self):
+    def test_validate_sample_duplicate_uprns(self):
```
Comment (AdamHawtin) [inline tests/test_validate_sample.py]: Same comment re `unique` UPRN's

**Review by AdamHawtin — APPROVED** [review]
👍 LGTM

**Pull request merged**